components/form-item.tsx

```diff
@@ -1,12 +1,23 @@
 import { ReactNode } from 'react'
+import clsx from 'clsx'
 
 export default function FormItem(props: {
   label: string
   children: ReactNode
+  direction?: 'horizontal' | 'vertical'
+  gap?: number
+  classNames?: string
```

Comment:
just use `className` here and do not rename it to `restCls`

Reply:
Fixed.